app/admin/rents.rb

```diff
@@ -0,0 +1,48 @@
+ActiveAdmin.register Rent do
+  permit_params :to
+
+  index do
+    selectable_column
+    id_column
+    column 'User' do |rent|
+      link_to(rent.user.email, admin_user_path(rent.user_id))
+    end
+    column :book
+    column :from
+    column :to
+    column :created_at
+    column :updated_at
+    actions
+  end
+
+  show do
+    attributes_table do
+      row 'User' do |rent|
+        link_to(rent.user.email, admin_user_path(rent.user_id))
+      end
+      row :book
+      row :from
+      row :to
+      row :created_at
+      row :updated_at
+    end
+  end
+
+  filter :user, collection: -> { User.all.map { |user| [user.email, user.id] } }
```

Comment:
just for your information, you are instantiating the whole active record object, when you only need the email and the id. You have the method pluck that only returns an array if the queried values. Check this post https://rubyinrails.com/2014/06/05/rails-pluck-vs-select-map-collect/